package.json

```diff
@@ -35,7 +35,14 @@
     "build:node": "npm run build:node:scripts && npm run build:node:views",
     "build": "npm run clean && npm run build:static && npm run build:node",
     "serve": "npm run build && node scripts/serve.js",
+    "test:js": "karma start",
     "test:node": "./scripts/test-node.sh",
     "test": "npm run test:node"
+  },
+  "devDependencies": {
+    "karma": "^1.7.1",
```

Comment:
This will probably need to be in `dependencies` rather than a `devDependencies` when you fix the comment above.

Reply:
Not sure why. Can you tell more?

Reply:
Oh, I just meant that binaries might not be created in `node_modules/.bin` if things are in `devDependencies` instead of `dependencies`, but if it's working then it's fine.